torch/csrc/jit/passes/quantization.cpp

```diff
@@ -101,20 +101,17 @@ void PropagateQuantInfo(std::shared_ptr<Graph>& graph) {
   throw std::runtime_error("Pass not implemented yet!");
 }
 
-static void addObserverFor(Value* v, Node* original_observer_node) {
-  Node* def = v->node();
+static Node* addObserverFor(Value* v, Node* original_observer_node, Node* def) {
```

Comment:
Name of argument `Node *def` is misleading - it is not a def (otherwise we wouldn't need it at all and would've just used `v->node()`). In fact, we don't need this at all - we only use it for two things:
1) `WithInsertPoint` - that could be just hoisted out of this function to the caller,
2) `def->owningGraph()` - we should just pass the graph directly, that would better reflect what's going.